src/styles.css

```diff
@@ -18,9 +18,25 @@ h1 {
   margin: 50px 0;
 }
 
-#tasks-container p {
+.hide {
+  display: none;
+}
+
+#tasks-container textarea {
   font-weight: 600;
-  max-width: 80%;
+  width: 65vw;
+  max-width: 340px;
+  font-family: Yuji Hentaigana Akebono, sans-serif;
+  overflow: auto;
+  resize: none;
+  outline: none;
+  border: none;
+  padding-top: 15px;
+  background: none;
+}
+
+.background {
+  background-color: blanchedalmond;
 }
 
 #list-container {
```

Comment:
[Optional changes]

- Please consider removing the bullets on the input and on the subtitle of the Todolist as they aren't necessary in this case